src/main/tut/kleisli-categories.md

```diff
@@ -0,0 +1,71 @@
+```Haskell
+type Writer a = (a, String)
+```
+```tut:silent
+type Writer[A] = (A, String)
+```
+................
+```Haskell
+a -> Writer b
+```
+```scala
+A => Writer[B]
+```
+................
+```Haskell
+(>=>) :: (a -> Writer b) -> (b -> Writer c) -> (a -> Writer c)
+```
+```scala
+def >=>[A, B, C](m1: A => Writer[B], m2: B => Writer[C]): A => Writer[C]
+```
+................
+```Haskell
+m1 >=> m2 = \x -> 
+    let (y, s1) = m1 x
+        (z, s2) = m2 y
+    in (z, s1 ++ s2)
+```
+```tut:silent
+object kleisli {
+  implicit class Fishy[A, B](m1: A => Writer[B]) {
+   def >=>[C](m2: B => Writer[C]): A => Writer[C] =
```

Comment:
This indeed does raise an interesting question. My gut feeling is that we shouldn't add noise like `implicit classes`, but otoh that's the only way to get an operator. Let's hear someone else's opinion as well

Reply:
I vote for rename `Fishy` , `KleisliOps` and add a comment like "to support better syntax." Since this usage pattern is typical in scala type classes, I think we should not shy away from using it.